R/conventional-commits.R

```diff
@@ -0,0 +1,4 @@
+# fix type commits should be translated to PATCH releases.
```

Comment:
that's a placeholder to hook this into `bump_version()` (some sort of automatic choice)